runner/src/main.rs

```diff
@@ -972,8 +972,9 @@ fn run_cargo_clean() -> Step {
     Step::Multiple {
         name: "cargo clean".to_string(),
         steps: crate_manifest_files()
+            .map(to_string)
             .map(|manifest_path| {
-                let manifest_path = manifest_path.to_str().unwrap();
+                let manifest_path = manifest_path.clone();
```

Comment:
This does not do anything useful :) You don't need this line anymore.

Reply:
Thank you for catching that!